Fruitties/shared/src/commonMain/kotlin/com/example/fruitties/utils/JsonConfig.kt

```diff
@@ -0,0 +1,22 @@
+/*
+ * Copyright 2024 The Android Open Source Project
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.example.fruitties.utils
+
+import kotlinx.serialization.json.Json
+
+object JsonConfig {
+    val json = Json { ignoreUnknownKeys = true }
+}
```

Comment:
nit: I'd say we can keep it in the `AppContainer` file and just use it without the object.